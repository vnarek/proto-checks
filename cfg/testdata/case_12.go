package main

func main() {
	x := new(int)
	max := new(int)
	*max = 5
	for i := *x; i < *max; i++ {
		{}
		{
			for j := *x; j < *max; j++ {
				{
					a = b
				}
			}
		}
		{}
	}
	x = nil
}