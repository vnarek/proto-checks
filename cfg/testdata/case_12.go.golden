[START]
  [x = alloc]
    [max = alloc]
      [i = *x]
        [j = *x]
          [connects to: [j = *x]]
          [a = b]
            [connects to: [j = *x]]
            [connects to: [a = b]]
            [x = null]
          [connects to: [x = null]]
        [connects to: [x = null]]
